app/codelab/install-packages.md

```diff
@@ -34,7 +34,7 @@ To verify that there are AngularUI packages available, use Bower to search for "
 bower search angular-ui-sortable
 ```
 
-There is one result for "angular-ui-sortable" so let’s install it along with [jQuery UI](http://jqueryui.com/) as we already have jQuery installed. To save you from searching, the package name for jQuery UI is "jquery-ui".
+There is one result for "angular-ui-sortable" which will also install [jQuery UI](http://jqueryui.com/) as we already have jQuery installed. If it didn't include JQuery UI the package is "jquery-ui".
```

Comment:
There are some things wrong in this sentence and this way it's kinda over-complicated. `angular-ui-sortable` has a hard dependency on jQuery UI, that has nothing to do with the fact that we already have `jquery` installed.

I'd suggest to make it something like this:

> There is one result, `angular-ui-sortable`, which will also install [jQuery UI](http://jqueryui.com/). The jQuery UI package is called `jquery-ui`.